code/game/objects/items/mail.dm

```diff
@@ -81,8 +81,8 @@
 		var/image/stamp_image = image(
 			icon = icon,
 			icon_state = stamp,
-			pixel_x = stamp_offset_x,
-			pixel_y = stamp_offset_y + bonus_stamp_offset
+			pixel_w = stamp_offset_x,
+			pixel_z = stamp_offset_y + bonus_stamp_offset
```

Comment:
you need to assign these directly because BYOND

Reply:
The bug with it was fixed, and the linters have been updated.
![image](https://github.com/user-attachments/assets/0005cb73-895d-446d-8db5-81c0ad059f10)

https://github.com/SpaceManiac/SpacemanDMM/commit/4b77cd487d0a7b6a069df20356b701af5b20489d

Just gotta wait on the new SpacemanDMM release with these.